imgqa/browseractions.py

```diff
@@ -537,3 +543,21 @@ def find_elements(self, locator):
                     value=locator['value']))
         else:
             AssertionError("Invalid locator type")
+
```

Comment:
👍  for the method addition. please merge this into older one removing suffix of `using_actions`